docs/source/virtual_rainforest/main_simulation.md

```diff
@@ -1,80 +1,158 @@
+  grid.
+
+### Simulation timescale
+
+The simulation runs between two dates with an update interval at which each science
+model is recalculated. These values are defined in the `core` configuration and are
+now validated to ensure that the start date, end date and update interval are sensible.
+
+```{note}
+The simulation uses 12 equal length months (30.4375 days) and equal length years (365.25
+days), ignoring leap years.
+```
+
+## Science models
 
-## Loading configuration files
+The Virtual Rainforest is implemented as model objects, each of which is responsible for
+simulating a particular aspect of the rainforest ecosystem. The models used for the
+specific simulation run can be set in the configuration. This will typically be the four
+standard models:
 
-As a first step, configuration files are loaded in and validated. These files are of
-`toml` format and are found based on paths provided by the user. When these files are
-loaded in they are validated to ensure that they are valid `toml`, and that all the
-required settings are present and not duplicated. The Virtual Rainforest code provides
-default settings for some configuration options and these will be used automatically if
-that setting is not found in the configuration, but if no default is set then the
-validation will fail. Further details can be found in the [configuration
-documentation](./core/config.md).
+* [`AbioticModel`](../api/abiotic.md),
+* `AnimalModel`,
+* `PlantModel` and
+* [`SoilModel`](../api/soil.md)
 
-## Grid creation
+but this can be [extended to include new models](../development/defining_new_models.md)
+or adopt different combinations of models.
 
-The next step is creating a grid, which defines the spatial structure of the simulation:
-the area, coordinate system and geometry of the individual cells that will be used in
-the simulation. The [grid documentation](./core/grid.md) describes this object further.
-This grid is then used to create an empty `Data` object of the correct size to describe
-the simulation.
+Once a list of models to configure has been extracted from the configuration, all
+science models run through a set of steps to prepare for the simulation to start. Each
+step is represented using a set of standard model methods that are run in the following
+order.
 
-## Loading and validation of input data
+### Model configuration
 
-This `Data` object now needs to be populated with data. The data used in the simulation
-is stored in a set of data files which are specified in the configuration. This data is
-then loaded using the `data.load_data_config` method, which has built in validation
-procedures to ensure that the loaded data can be mapped onto the spatial grid and also
-other core dimensions for the configured simulation. Further details can be found in the
-[data system](./core/data.md) and [core axes](./core/axes.md) documentation.
+The core configuration will include the configuration details for each individual
```

Comment:
These details won't be in the core configuration

Reply:
Yeah - clumsily phrased.